pyvista/plotting/_property.py

```diff
@@ -107,6 +108,11 @@ class Property(_vtk.vtkProperty):
         * ``"front"`` - Enable frontface culling
         * ``'none'`` - Disable both backface and frontface culling
 
+    edge_opacity : float, default: :attr:`pyvista.plotting.themes.Theme.edge_opacity`
+        Edge opacity of the mesh. A single float value that will be applied globally
+        edge opacity of the mesh and uniformly applied everywhere - should be
+        between 0 and 1.
```

Comment:
Would you please add a note here that this is only available for VTK 9.3+

Reply:
```suggestion
    edge_opacity : float, default: :attr:`pyvista.plotting.themes.Theme.edge_opacity`
        Edge opacity of the mesh. A single float value that will be applied globally
        edge opacity of the mesh and uniformly applied everywhere - should be
        between 0 and 1.

        .. note::
            `edge_opacity` uses ``SetEdgeOpacity`` as the underlying method which
            requires VTK version 9.3 or higher. If ``SetEdgeOpacity`` is not
            available, `edge_opacity` is set to 1.
```